config/core/test-configs-chromeos.yaml

```diff
@@ -314,6 +342,11 @@ test_configs:
       - cros-tast-perf
       - cros-tast-platform
       - cros-tast-video
+    
+  - device_type: asus-C523NA-A20057-coral_chromeos
+    test_plans:
+      - cros-baseline-fixed
+      - cros-flash
```

Comment:
Can we replace test plans by updated ones?
```
    test_plans:
      - cros-baseline
      - cros-baseline-fixed
      - cros-tast-kernel
      - cros-tast-perf
      - cros-tast-platform
      - cros-tast-video
```

Reply:
Okay. Let me test them first.

Reply:
@nuclearcat  I have updated the configurations to use `chromebook-generic-params` and test plans.